assets/src/edit-story/karma/fixture.js

```diff
@@ -382,23 +383,29 @@ class APIProviderFixture {
         []
       );
 
-      const getAllFonts = useCallback(() => {
+      const getAllFonts = useCallback(
         // @todo: put actual data to __db__/
-        return asyncResponse(
-          [TEXT_ELEMENT_DEFAULT_FONT].map((font) => ({
-            name: font.family,
-            value: font.family,
-            ...font,
-          }))
-        );
-      }, []);
-
-      // eslint-disable-next-line no-unused-vars
-      const getMedia = useCallback(({ mediaType, searchTerm, pagingNum }) => {
+        () =>
+          asyncResponse(
+            [TEXT_ELEMENT_DEFAULT_FONT].map((font) => ({
+              name: font.family,
+              value: font.family,
+              ...font,
+            }))
+          ),
+        []
+      );
+
+      const getMedia = useCallback(
         // @todo: arg support
-        // @todo: put actual data to __db__/
-        return asyncResponse({ data: [], headers: {} });
-      }, []);
+        // eslint-disable-next-line no-unused-vars
+        ({ mediaType, searchTerm, pagingNum }) =>
+          asyncResponse({
```

Comment:
Just to be safe, could you please check that `pageNum === 1` and other args are falsy (or whatever expected) and throw a "not yet available" error if not?

Reply:
We coudl also do something with a `spyOn`:

```
const getMedia = jasmine.createSpy('getMedia');
getMedia
  .withArgs({mediaType: 'ALL', searchTerm: '', pagingNum: 1})
  .and.returnValue(asyncResponse(...));
```

That way we can add a few variants without lots of ifs.

Reply:
I'll just implement the expected functionality with the actual dummy content - it's fairly simple.

Reply:
Like 76a227281 - doesn't that seem to do it? It definitely works. Even though paging isn't actually used because the "num pages" header is faked to always say 1.